lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
+  label = `${label}`;
   this._times.set(label, process.hrtime());
```

Comment:
Personally, I would just use the template literal in this line and spare the assignment above.